src/mmrelay/main.py

```
@@ -242,9 +243,15 @@ async def shutdown():
                         matrix_logger.warning(
                             "Matrix sync_forever completed unexpectedly"
                         )
-                    except Exception:  # noqa: BLE001 — sync loop must keep retrying
-                        # Log the exception and continue to retry
-                        matrix_logger.exception("Matrix sync failed")
+                    except (
+                        Exception
```

Ruff (0.14.6)

Do not catch blind exception: `Exception`

(BLE001)